www/install/var/configFileGorgoneTemplate

```diff
@@ -0,0 +1,66 @@
+name: gorgoned-central
+description: Configuration for Central server
+database:
+  db_centreon:
+    dsn: "mysql:host=--ADDRESS--:--DBPORT--;dbname=--CONFDB--"
+    username: "--DBUSER--"
+    password: "--DBPASS--"
+  db_centstorage:
+    dsn: "mysql:host=--ADDRESS--:--DBPORT--;dbname=--STORAGEDB--"
+    username: "--DBUSER--"
+    password: "--DBPASS--"
+gorgonecore:
+  privkey: "--CENTREON_SPOOL--/.gorgone/privkey.pm"
```

Comment:
Maybe something like "rsakey.priv.pem" for the key name (like for the public one)

Reply:
indeed